spec/requests/api/v1/events/events_spec.rb

```diff
@@ -20,3 +22,50 @@
     )
   end
 end
+
+describe 'POST /v1/events' do
+  it 'saves the address, lat, lon, name, and started_at date' do
+    date = Time.current
```

Comment:
Have use used FactoryGirl to build a event?

`event = build :event`

In the factory it's going to create an Event with some nice values ... it's a good idea to take advantage of that so don't need to think up values for the attributes each time.